### PR TITLE
Audio engine audio state delegate

### PR DESCRIPTION
**Pull request by JB184351**
Got rid of some commented code.

### CODE REVIEW

File: TalkAloud/TalkAloud/AudioViewController.swift [hunk 1]
```diff
@@ -9,16 +9,16 @@
 import UIKit
 import AVFoundation
 
-class AudioViewController: UIViewController {
-    
+class AudioViewController: UIViewController, AudioEngineStateChangeDelegate {
     // Intialized AudioEngine object so properties and methods can be used for later
     var audioEngine = AudioEngine()
-    
+    var delegate: AudioEngineStateChangeDelegate?
```
Comment (virgil-adumitroae) [inline TalkAloud/TalkAloud/AudioViewController.swift]: Why this here?

File: TalkAloud/TalkAloud/AudioEngine.swift [hunk 2]
```diff
@@ -9,7 +9,12 @@
 import Foundation
 import AVFoundation
 
+protocol AudioEngineStateChangeDelegate: class {
+    func didUpdateAudioState(willSetAudioEngineStateTo audioState: AudioEngineState)
```
Comment (virgil-adumitroae) [inline TalkAloud/TalkAloud/AudioEngine.swift]: You can rename to `func didUpdateAudioState(with audioState: AudioEngineState)`

**Review by virgil-adumitroae — CHANGES_REQUESTED** [review]

File: TalkAloud/TalkAloud/AudioViewController.swift (hunk 1 above)
Comment (JB184351) [reply TalkAloud/TalkAloud/AudioViewController.swift]: Good question. I didn't need it there just made a mistake when I was implementing the delegate method and I thought I needed it.

File: TalkAloud/TalkAloud/AudioEngine.swift (hunk 2 above)
Comment (JB184351) [reply TalkAloud/TalkAloud/AudioEngine.swift]: Yeah, I thought of that when I pushed up my code haha.

**Review by virgil-adumitroae — APPROVED** [review]

**Pull request merged**